app/models/concerns/translatable.rb

```diff
@@ -29,6 +29,10 @@ module ClassMethods
     #   set_translation('foo', text, locale: :en)
```

Comment:
Can these items above be removed? There seems to be a lot commented out in this file.